ui/src/views/Releases/ListReleaseRevisions/index.jsx

```diff
@@ -64,7 +64,7 @@ function ListReleaseRevisions(props) {
 
   useEffect(() => {
     if (fetchedRelease.data) {
-      fetchRevisions(releaseName, fetchedRelease.data.data.product);
+      fetchRevisions(releaseName, 1);
```

Comment:
I assume this was intentional but just in case, doesn't hurt to bring it up :)

Reply:
Yup. The interface to `getRevisions` now requires the api version to be passed, and doesn't need product: https://github.com/mozilla-releng/balrog/blob/5324a2fb49e0e650ca1e2fe26785449a87a2df6f/ui/src/services/releases.js#L40